### ISSUE TITLE
update mac db make app crash

### ISSUE THREAD

**Issue by phra** [roccomuso/kickthemout#2]
hi.

if from the gui i select 'update mac db', it crashes with this message:

```
? What to do? Update MAC Vendors DB
(node:13554) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): TypeError: chalk.orange is not a function
```

OS: debian testing

**Comment by roccomuso** [comment]
Thanks. Bug fixed in [65510ffb](https://github.com/roccomuso/kickthemout/commit/65510fb7d461d86fb2b1e0fed36bc3126e81077b) and new version v1.1.1 published on npm.

**Comment by phra** [comment]
you welcome.
